third_party/blink/renderer/bindings/generated_in_modules.gni

```diff
@@ -2791,41 +2791,6 @@ generated_interface_sources_in_modules = [
   "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_xr_webgl_sub_image.h",
 ]
 
-if (is_cobalt) {
```

Comment:
FYI moved this to the bottem of the file because generated_union_sources_in_modules is not declared here and I needed to add it to if (is_cobalt)

Reply:
Hmm, you can always add another "is_cobalt" in union_sources, if needed. I suspect that's a better approach as moving the whole stanza is likely harder when we do a rebase. @yell0wd0g to confirm.